### ISSUE TITLE
Fade in annotations [improvement]

### ISSUE THREAD

**Issue by chuckdries** [pisi/Reel#180]
It would be much nicer looking if the annotations faded in smoothly rather than just appear. Maybe look into using the jquery fadetoggle() function for this

**Comment by pisi** [comment]
Yeah, you're right, it would be much nicer. However am worried about performance of this especially of queueing when toggled rapidly (when grabbed and thrown for example). Have to do some tests, but I will definitely try it!

**Comment by chuckdries** [comment]
Maybe you can make it an option so fadeannotations: true will make them fade in, that way if you're not worried about performance or you know you're using a high end machine (like I am, I'm  using the plugin for a walk up informational kiosk) you can use it. Make it false by default

**Comment by pisi** [comment]
OK, sounds reasonable. Scheduled for the next release.

**Comment by chuckdries** [comment]
Thanks! Keep up the great work!

**Comment by JWestarp** [comment]
What about this? Is there a fade available now?
